Documentation/Diagnostics/UsingVisualProfiler.md

```diff
@@ -32,6 +32,9 @@ The specific platform and hardware configuration will play a significant role in
 - Microsoft HoloLens: 60
 - Windows Mixed Reality Ultra: 90
 
+> [!NOTE]
+> Due to [frame rate throttling on HoloLens when default MRC is active](https://docs.microsoft.com/windows/mixed-reality/mixed-reality-capture-for-developers#what-to-expect-when-mrc-is-enabled-on-hololens), the visual profiler hides itself while videos and photos are captured.
```

Comment:
There is the edge case where a user want's to capture the profiler with MRC? But, I don't think we need to add an option for that unless it is brought up by the community.

Reply:
That was my thought too. I could add it to the profile if we want though.

Reply:
Added as a setting both on the MonoBehaviour and the profile